cidc_api/models/files/details.py

```diff
@@ -289,6 +279,31 @@ class FileDetails(NamedTuple):
         "the log file produced by the Salmon analysis",
         "A plain-text file containing the time and output of all logging by Salmon during sample analysis.",
     ),
+    "/rna/analysis/microbiome/_addSample_report.txt": FileDetails(
```

Comment:
won't this just be "/rna/analysis/microbiome/sample_report.txt"? see https://github.com/CIMAC-CIDC/cidc-schemas/blob/master/cidc_schemas/schemas/assays/components/ngs/rna/microbiome.json

Reply:
fixed @scvannost !